django/core/management/commands/shell.py

```diff
@@ -111,10 +127,68 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
+    def get_and_report_namespace(self, verbosity, no_imports=False):
+        if no_imports:
+            return {}
+
+        namespace = self.get_namespace()
+
+        if verbosity < 1:
+            return namespace
+
+        amount = len(namespace)
+        msg = f"{amount} objects imported automatically"
```

Comment:
```suggestion
        msg = f"{len(namespace)} objects imported automatically"
```
`amount` is used only here

Reply:
This goes against the [Django code style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), a variable has to be defined to be used in the f-string for non-trivial values:

> f-strings should use only plain variable and property access, with prior local variable assignment for more complex cases

cc/ @sarahboyce @salvo-polizzi

Reply:
It seems to me a trivial use, but you're right that in the coding style guidelines it explicitly stated the opposito.